addon/webextension/sitehelper.js

```diff
@@ -0,0 +1,41 @@
+/* globals catcher, callBackground */
+/** This is a content script added to all screenshots.firefox.com pages, and allows the site to
+    communicate with the add-on */
+
+window.sitehelper = (function () {
```

Comment:
I'd rather we use export helpers (`exportFunction`, `cloneInto`, ...) than custom events for this.